pkg/controller/nmstate/nmstate_controller_test.go

```diff
@@ -69,4 +69,32 @@ var _ = Describe("NMstate controller reconcile", func() {
 			Expect(result).To(Equal(reconcile.Result{}))
 		})
 	})
+	Context("node selector is set ", func() {
+		var (
+			request reconcile.Request
+		)
+		BeforeEach(func() {
+			s := scheme.Scheme
+			s.AddKnownTypes(nmstatev1alpha1.SchemeGroupVersion,
+				&nmstatev1alpha1.NMstate{},
```

Comment:
maybe we have to add DaemonSet here so we can tretrieve it later to heck nodeSelctor ?